### PR TITLE
Remove unitary inverse

### LINKED ISSUE #49: Remove unitary_inverse from quadratic extension
On a quadratic extension field, `x.unitary_inverse() = x.clone().conjugate()`. The method is barely used, its only used in cyclotomic exponentiation. 

imo, it feels like we shouldn't be using the name `unitary_inverse` and conjugate. Having both methods is just API bloat / confusion. I'd suggest deleting it, or alternatively renaming them to `conjugate()` and `conjugate_in_place()`.

### PR DESCRIPTION
**Pull request by ValarDragon**
Closes #49

### CODE REVIEW

File: ff/src/fields/mod.rs [hunk 1]
```diff
@@ -107,7 +107,8 @@ pub trait Field:
 {
     type BasePrimeField: PrimeField;
 
-    /// Returns the characteristic of the field.
+    /// Returns the characteristic of the field,
+    /// in big-endian representation.
```
Comment (Pratyush) [inline ff/src/fields/mod.rs:111]:
```suggestion
    /// in little-endian representation.
```

**Comment by Pratyush** [comment]
btw can you confirm that `curves` compiles with this change? You can follow the steps here: https://github.com/arkworks-rs/algebra/issues/34

File: ff/src/fields/mod.rs (hunk 1 above)
Comment (ValarDragon) [reply ff/src/fields/mod.rs:111]: ah you're right, got them mixed up 😅

**Comment by ValarDragon** [comment]
It still compiles (assuming I did the instructions in #34 correctly)

**Review by Pratyush — APPROVED** [review]

**Pull request merged**